src/__tests__/gilded_rose_spec.js

```diff
@@ -1,11 +1,114 @@
-import { Shop, Item } from '../gilded_rose';
+import { Shop, Item } from "../gilded_rose";
 
-describe("Gilded Rose", function () {
+describe("Gilded Rose", function() {
+  describe("Item", () => {
+    it("initiates correctly", () => {
+      const item = new Item("foo", 10, 40);
+      expect(item).toEqual(new Item("foo", 10, 40));
+    });
+  });
 
-    it("should foo", function () {
-        const gildedRose = new Shop([new Item("foo", 0, 0)]);
-        const items = gildedRose.updateQuality();
-        expect(items[0].name).toEqual("fixme");
+  describe("Shop", () => {
+    it("stores given items", () => {
+      const item = new Item("foo", 10, 40);
+      const shopItems = [item, item];
+      const shop = new Shop(shopItems);
+      expect(shop.items).toEqual(shopItems);
     });
 
+    describe("updateQuality", () => {
+      it("lowers sellIn and quality on each run", () => {
```

Comment:
What is the sellIn and the quality? By only reading this test, imo it's not clear to me

Reply:
What does 

> on each run

mean?